configs/services/dynamodb.json

```
@@ -2,6 +2,7 @@
   "libraryName": "amazonka-dynamodb",
   "extraDependencies": [
     "aeson",
+    "amazonka-dynamodb-attributevalue >= 2.0",
```

Normally I'd be in favour of bounds everywhere, but because everything in amazonka-land moves in lockstep I think we just apply an unbounded dep here:
```suggestion
    "amazonka-dynamodb-attributevalue",
```